notebooks/official/training/multi_node_ddp_nccl_vertex_training_with_custom_container.ipynb

```diff
@@ -125,252 +125,193 @@
       ]
     },
     {
       "cell_type": "code",
       "execution_count": null,
       "metadata": {
-        "id": "254614fa0c46"
+        "id": "3989ed5bea75"
       },
       "outputs": [],
       "source": [
-        "# ! gcloud auth login"
+        "PROJECT_ID = \"[your-project-id]\"  # @param {type:\"string\"}\n",
+        "LOCATION = \"us-central1\"  # @param {type:\"string\"}"
       ]
     },
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "ef21552ccea8"
+        "id": "zgPO1eR3CYjk"
       },
       "source": [
-        "**3. Colab**\n",
-        "<br>Uncomment and run the following code:"
+        "### Create a Cloud Storage bucket\n",
+        "\n",
+        "Create a storage bucket to store intermediate artifacts such as datasets."
       ]
     },
     {
       "cell_type": "code",
       "execution_count": null,
       "metadata": {
-        "id": "603adbbf0532"
+        "id": "MzGDU7TWdts_"
       },
       "outputs": [],
       "source": [
-        "# from google.colab import auth\n",
-        "\n",
-        "# auth.authenticate_user()"
-      ]
-    },
-    {
-      "cell_type": "markdown",
-      "metadata": {
-        "id": "c13224697bfb"
-      },
-      "source": [
-        "**4. Service account or other**\n",
-        "* See how to grant Cloud Storage permissions to your service account at https://cloud.google.com/storage/docs/gsutil/commands/iam#ch-examples."
+        "BUCKET_URI = f\"gs://your-bucket-name-{PROJECT_ID}-unique\"  # @param {type:\"string\"}"
       ]
     },
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "zgPO1eR3CYjk"
+        "id": "create_bucket"
       },
       "source": [
-        "### Create a Cloud Storage bucket\n",
-        "\n",
-        "Create a storage bucket to store intermediate artifacts such as datasets."
+        "**If your bucket doesn't already exist**: Run the following cell to create your Cloud Storage bucket."
       ]
     },
     {
       "cell_type": "code",
       "execution_count": null,
       "metadata": {
-        "id": "MzGDU7TWdts_"
+        "id": "NIq7R4HZCfIc"
       },
       "outputs": [],
       "source": [
-        "BUCKET_URI = f\"gs://your-bucket-name-{PROJECT_ID}-unique\"  # @param {type:\"string\"}"
+        "! gsutil mb -l {LOCATION} -p {PROJECT_ID} {BUCKET_URI}"
       ]
     },
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "create_bucket"
+        "id": "306bf9ab7f7f"
       },
       "source": [
-        "**Only if your bucket doesn't already exist**: Run the following cell to create your Cloud Storage bucket."
+        "### Initialize Vertex AI SDK for Python\n",
+        "\n",
+        "To get started using Vertex AI, you must have an existing Google Cloud project and [enable the Vertex AI API](https://console.cloud.google.com/flows/enableapi?apiid=aiplatform.googleapis.com)."
```

Comment:
Change link text. Remove "enable the"
- "you must have an existing Google Cloud project and enable the [Vertex AI API]..."